examples/terraform/azure/output.tf

```diff
@@ -22,6 +22,21 @@ output "kubeone_api" {
   }
 }
 
+# Hack to ensure we get access to public ip in first attempt
+resource "time_sleep" "wait_30_seconds" {
+  depends_on = [azurerm_virtual_machine.control_plane]
+  create_duration = "30s"
+}
+
+data "azurerm_public_ip" "control_plane" {
+  depends_on = [
+    time_sleep.wait_30_seconds
+  ]
+  count = var.control_plane_vm_count
+  name                = "${var.cluster_name}-cp-${count.index}"
+  resource_group_name = azurerm_resource_group.rg.name
+}
```

Comment:
Can you run [`terraform fmt`](https://www.terraform.io/docs/cli/commands/fmt.html) to format the file?